man/man1/git-hack.1

```diff
@@ -1,7 +1,7 @@
 .TH "GIT-HACK" "1" "04/02/2015" "Git Town 0\&.6\&.0" "Git Town Manual"
 
 .SH "NAME"
-git hack \- cut a new feature branch off the main branch
+git hack \- create a new feature branch
 
 .SH "SYNOPSIS"
 \fIgit hack\fR <branchname> [parent branch name]
```

Comment:
I believe angled brackets only imply user input. I believe if we fed this into docopt the `parent branch name` cannot be any string but would need to be exactly `parent branch name`.